### PR TITLE
Lesson 18 Wiki Notes

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: Class-18-(ORM-Part-3).md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+## In the prep work this week, the students learned
+
+1. How to build persistent model classes with one-to-one and many-to-many relationships.
+1. What a composite key is.
+1. What a join table is.
+1. How to use different annotations such as ``@OneToOne`` and ``@ManyToMany``.
+
+## Announcements
+
+1. Assignment #4 is due soon!
```
Comment (CarlyLanglois) [inline Class-18-(ORM-Part-3).md]: I would also say that with today's material, they have all they need to complete it.

File: Class-18-(ORM-Part-3).md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+1. Check with your course manager for any additional announcements.
+
+## Large Group Time (Instructor)
+
+### Class 18 Topics That Require Careful Attention
+1. Review the one-to-many and many-to-one relationships in preparation to one-to-one and many-to-many relationships.
```
Comment (CarlyLanglois) [inline Class-18-(ORM-Part-3).md]: in preparation to discuss

File: Home.md [hunk 3]
```diff
@@ -30,4 +30,6 @@ Fork the [wiki repository](https://github.com/LaunchCodeEducation/java-web-devel
 * [Class 13 - Work Day](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-13-(Work-Day))
 * [Class 14 - Databases and SQL](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-14-(SQL-Part-1))
 * [Class 15 - The SQL Sequel](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-15-(SQL-Part-2))
-* [Class 16 - Introduction to Object-Relational Mapping](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-16-(ORM-Part-1))
\ No newline at end of file
+* [Class 16 - Introduction to Object-Relational Mapping](https://github.com/LaunchCodeEducation/java-web-development/wiki/Class-16-(ORM-Part-1))
+
+* [Class 18 - ORM Part 3]()
```
Comment (CarlyLanglois) [inline Home.md]: oops add this link!

**Review by CarlyLanglois — APPROVED** [review]
good to go after a merge with master and a few comments

File: Class-18-(ORM-Part-3).md (hunk 1 above)
Comment (gildedgardenia) [reply Class-18-(ORM-Part-3).md]: Done!

File: Class-18-(ORM-Part-3).md (hunk 2 above)
Comment (gildedgardenia) [reply Class-18-(ORM-Part-3).md]: Done!

File: Home.md (hunk 3 above)
Comment (gildedgardenia) [reply Home.md]: Done!

**Pull request merged**